tools/Scripts/SymbolicLinkHelpers.psm1

```diff
@@ -70,6 +70,46 @@ Function FixSymbolicLinksForDirectory {
 	Write-Output "Fixing symbolic links for the following directory: $Directory"
 	$Directory | dir -Recurse -File | ?{$_.LinkType -eq "SymbolicLink" } | Restore-SymbolicLink
 
+	# If any links were created before the repository was configured to use
+	# symlinks, those links need to be restored.
+	git status --porcelain | Restore-SymbolicLinkFromTypeChange
+    }
+}
+
+Function HardCopySymbolicLink {
+    [CmdletBinding()]
+    Param(
+        [Parameter(ValueFromPipeline)]$file
+    )
+
+    Process {
+        $fileContent = $file.Target
+        Write-Output "Copying content from $fileContent to "$file.FullName
+
+        $currDir = Get-Location
+        $fileDir = Split-Path $file.FullName
+        $fileName = Split-Path $file.FullName -Leaf
+
+        Set-Location $fileDir
+        (Get-Item $file).Delete()
+        New-Item -ItemType Directory -Force -Path $fileName
+        Copy-Item -Path "$fileContent\*" -Destination $fileName -Recurse
+
+        Set-Location $currDir
```

Comment:
Does this work for both file and directory symlinks? If it's intended for both, maybe a comment + naming the variables here $fileOrDirectory would make that clearer

Reply:
Good question, we have not had any file symlinks to test with. This naming was based on the previous piped arguments higher up, so i will probably keep this as it is for now.